qualcommax: ipq6018 rework nss_port5 clock to multiple conf

dr-m opened the pull request:
This is a port of the gcc-ipq8074 patch https://lore.kernel.org/all/20231220221724.3822-4-ansuelsmth@gmail.com/ to the ipq6018 SoC.

This fixes intermittent `dmesg` errors of the following form:
```
nss_port5_rx_clk_src: rcg didn't update its configuration.
```
This was originally contributed by @JiaY-shi; see #14950 and https://forum.openwrt.org/t/qualcommax-nss-port5-rx-clk-src-rcg-didnt-update-its-configuration/232618/4.

I had this patch running for 31 days and 10 hours on my gl.Inet GL-AX1800 router with no issues. I just rebuilt and installed this version on the current OpenWrt version.

aiamadeus commented:
<img width="861" height="165" alt="image" src="https://github.com/user-attachments/assets/0137cf2f-0517-4153-8af4-f8903c771575" />

dr-m committed:
qualcommax: ipq6018 rework nss_port5 clock to multiple conf

This fixes intermittent dmesg errors
"nss_port5_rx_clk_src: rcg didn't update its configuration."

Signed-off-by: Marko Mäkelä <marko.makela@iki.fi>
Link: https://github.com/openwrt/openwrt/pull/19620
Signed-off-by: Robert Marko <robimarko@gmail.com>
